code/index.html

```diff
@@ -3,28 +3,231 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylsheet" href="./normalize.css" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+    />
+    <link rel="icon" type="img/png" href="./img/flower.png">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="https://www.gordonmac.com/wp-content/themes/2016/vendor/vide/jquery.vide.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!-- GOOGLE FONTS -->
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Sacramento&display=swap');
+      </style>
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Playfair+Display&family=Sacramento&display=swap');
+  </style>
```

Comment:
If you're using @import to get a font then it should be at the top of the CSS file. The other solution is to use the link and put in the `<head>` instead!